wlutil/launch.py

```diff
@@ -97,39 +135,76 @@ def launchWorkload(baseConfig, jobs=None, spike=False, interactive=True):
     if not spike and baseConfig.get('qemu', True) is None:
         raise RuntimeError("This workload does not support qemu")
 
+    if not spike:
+        startVDE()
+
     if jobs is None:
         configs = [baseConfig]
     else:
         configs = [baseConfig['jobs'][j] for j in jobs]
 
     baseResDir = wlutil.getOpt('res-dir') / wlutil.getOpt('run-name')
+
+    screenIdentifiers = {}
+
+    try:
+        count = 0;
+        for config in configs:
+            if config['launch']:
+                
+                count += 1
```

Comment:
it might be clearer to name this something like "jobSlot" or "jobIdx" or something like that.